knowledge-base/grid-change-font-size.md

```diff
@@ -23,11 +23,13 @@ res_type: kb
 
 ## Description
 
-How to change the Grid font size? How to set custom font styles?
+* How to change the Grid font size? 
+* How to set custom font styles?
 
 ## Solution
 
-[Override the theme styles]({%slug themes-override%}) and set the desired font size to the Grid and its child components.
+1. Set a custom CSS class to the Grid through the `Class` parameter. This configuration will allow you to target specific Grid instances.
```

Comment:
Switch to present simple tense as per our Style Guide recommendations [here](https://docs.telerik.com/style-guide/basic-rules-and-guidelines#use-simple-tenses).